src/helpers.js

```diff
@@ -71,10 +71,14 @@ export function randomChar() {
  * @return {Document}
  */
 export async function fresh(document) {
-  const Model = document.model;
-  const hashKey = Model.schema.getHashKey();
+  const model = document.model;
 
-  return Model.get(document[hashKey]);
+  // Figure out what schema we use for this document:
+  const schema = await model.schemaForObject(document);
+  const hashKey = schema.getHashKey();
```

Comment:
This broke because of an (internal) API that we're relying on to be able to replicate this `$model->fresh()` behavior (which we only use in tests). Since Dyanmoose now supports multiple schemas per model, we have to ask for the correct schema for the given object, rather than simply reading it off `model.schema`.